dawn/src/dawn/Optimizer/PassInlining.cpp

```diff
@@ -117,23 +117,20 @@ class Inliner : public iir::ASTVisitor {
     if(AccessIDOfCaller_ == 0) {
       // We are *not* called within an arugment list of a stencil function, meaning we can store the
       // return value in a local variable.
-      int AccessID = instantiation_->nextUID();
-      auto returnVarName = iir::InstantiationHelper::makeLocalVariablename(
-          curStencilFunctioninstantiation_->getName(), AccessID);
 
-      newExpr_ = std::make_shared<iir::VarAccessExpr>(returnVarName);
-      auto newStmt =
-          iir::makeVarDeclStmt(dawn::Type(BuiltinTypeID::Float, CVQualifier::Const), returnVarName,
-                               0, "=", std::vector<std::shared_ptr<iir::Expr>>{stmt->getExpr()});
+      // Declare and register the variable
```

Comment:
I'd prefer passing a local var `keepVarName = false` instead of just `false` here, but maybe that's overly verbose.

Reply:
Agree, makes the code more readable.